server/api/25words.js

```diff
@@ -63,13 +63,14 @@ router.post("/", async (req, res, next) => {
 
   //loop through the random index array
   for (let i = 0; i < randomWordsIndexArray.length; i++) {
-    console.log(allWords[randomWordsIndexArray[2]].dataValues.word);
     //assign the last number in layout array as the team number
     const teamNumber = layout.pop();
     const word = {
       //change this if front end needs more than the word itself
       word: allWords[randomWordsIndexArray[i]].dataValues.word,
       isVisibleToAll: false,
+      //so react return item can have a unique key={word.id}
+      id: allWords[randomWordsIndexArray[i]].dataValues.id,
```

Comment:
I see - so you are using the Word Model's id for the card we are passing up.

eg, it looks like:
```
[
  { word: 'Screen', isVisibleToAll: false, id: 309, teamNumber: 1 },
  { word: 'Plastic', isVisibleToAll: false, id: 269, teamNumber: 1 },
  { word: 'Genius', isVisibleToAll: false, id: 144, teamNumber: 1 },
  //............
];
```


I am thinking about how, once the cards are on the front end, we may actually want to use these ids to reference of where they are on the board, eg they are the 5th card, or the 19th card.
So I think it might be better just to use `i` from this loop. so it looks like->
```js
id: i,
```
so that it can looks like:
```
[
  { word: 'Screen', isVisibleToAll: false, id: 0, teamNumber: 1 },
  { word: 'Plastic', isVisibleToAll: false, id: 1, teamNumber: 1 },
  { word: 'Genius', isVisibleToAll: false, id: 2, teamNumber: 1 },
  //............
];
```

Reply:
> I see - so you are using the Word Model's id for the card we are passing up.
> 
> eg, it looks like:
> 
> ```
> [
>   { word: 'Screen', isVisibleToAll: false, id: 309, teamNumber: 1 },
>   { word: 'Plastic', isVisibleToAll: false, id: 269, teamNumber: 1 },
>   { word: 'Genius', isVisibleToAll: false, id: 144, teamNumber: 1 },
>   //............
> ];
> ```
> 
> I am thinking about how, once the cards are on the front end, we may actually want to use these ids to reference of where they are on the board, eg they are the 5th card, or the 19th card. So I think it might be better just to use `i` from this loop. so it looks like->
> 
> ```js
> id: i,
> ```
> 
> so that it can looks like:
> 
> ```
> [
>   { word: 'Screen', isVisibleToAll: false, id: 0, teamNumber: 1 },
>   { word: 'Plastic', isVisibleToAll: false, id: 1, teamNumber: 1 },
>   { word: 'Genius', isVisibleToAll: false, id: 2, teamNumber: 1 },
>   //............
> ];
> ```

if we want to refer to their location on board, can we just use 25wordsArray.indexOf(word)? I want to keep the index and id separate, just in case we need to use both values at a certain point.